{ 'stand_up_back_100': { 'head': [],
                         'left_arm': [ { 'duration': 2.02,
                                         'name': u'stand_up_back_100',
                                         'positions': [ 0.1018,
                                                        -0.9632,
                                                        0.0,
                                                        0.0007,
                                                        0.0227,
                                                        0.0007,
                                                        0.0007],
                                         'starttime': 0.0},
                                       { 'duration': 1.97,
                                         'name': u'stand_up_back_200',
                                         'positions': [ -2.356,
                                                        0.8033,
                                                        0.001,
                                                        0.0068,
                                                        -0.0021,
                                                        -1.5711,
                                                        -0.001],
                                         'starttime': 13.015}],
                         'left_leg': [ { 'duration': 2.021,
                                         'name': u'stand_up_back_100',
                                         'positions': [ -0.0018,
                                                        0.0009,
                                                        -0.9316,
                                                        0.0033,
                                                        -0.0005,
                                                        -0.0001],
                                         'starttime': 0.0},
                                       { 'duration': 1.969,
                                         'name': u'stand_up_back_200',
                                         'positions': [ 0.0016,
                                                        -0.0125,
                                                        -1.5008,
                                                        1.7741,
                                                        -1.0194,
                                                        0.0043],
                                         'starttime': 2.021},
                                       { 'duration': 2.02,
                                         'name': u'stand_up_back_300',
                                         'positions': [ -0.0036,
                                                        0.0052,
                                                        -1.5735,
                                                        2.8034,
                                                        -0.1148,
                                                        0.0052],
                                         'starttime': 3.99}],
                         'right_arm': [ { 'duration': 2.021,
                                          'name': u'stand_up_back_100',
                                          'positions': [ -0.1018,
                                                         0.9632,
                                                         -0.0,
                                                         -0.0007,
                                                         -0.0227,
                                                         -0.0007,
                                                         -0.0007],
                                          'starttime': 0.0},
                                        { 'duration': 1.97,
                                          'name': u'stand_up_back_200',
                                          'positions': [ 2.356,
                                                         -0.8033,
                                                         -0.001,
                                                         -0.0068,
                                                         0.0021,
                                                         1.5711,
                                                         0.001],
                                          'starttime': 13.015}],
                         'right_leg': [ { 'duration': 1.969,
                                          'name': u'stand_up_back_100',
                                          'positions': [ 0.0018,
                                                         -0.0009,
                                                         0.9316,
                                                         -0.0033,
                                                         0.0005,
                                                         0.0001],
                                          'starttime': 6.027},
                                        { 'duration': 1.97,
                                          'name': u'stand_up_back_200',
                                          'positions': [ -0.0016,
                                                         0.0125,
                                                         1.5008,
                                                         -1.7741,
                                                         1.0194,
                                                         -0.0043],
                                          'starttime': 7.996},
                                        { 'duration': 2.071,
                                          'name': u'stand_up_back_300',
                                          'positions': [ 0.0036,
                                                         -0.0052,
                                                         1.5735,
                                                         -2.8034,
                                                         0.1148,
                                                         -0.0052],
                                          'starttime': 9.966000000000001}],
                         'torso': [ { 'duration': 1.001,
                                      'name': u'stand_up_back_100',
                                      'positions': [0.001, 0.0085],
                                      'starttime': 11.989}]}}
